lib/config/defaults.js

```diff
@@ -1082,7 +1102,9 @@ const applyOutputDefaults = (
 	F(
 		environment,
 		"bigIntLiteral",
-		() => /** @type {boolean | undefined} */ (tp && tp.bigIntLiteral)
+		() =>
+			tp &&
+			optimistic(/** @type {boolean | undefined} */ (tp && tp.bigIntLiteral))
```

Comment:
We use the same for `const`/`forOf`/etc, so let's use the same here when no targets avaliable